index.js

```diff
@@ -41,7 +50,19 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
```

Comment:
parseInt() works for us here in this function but I it's important to know that it will NOT return any decimals in the case we have those.